apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -8,17 +8,38 @@ import {createVideoWithFallback} from '@cdo/apps/code-studio/videos';
 import $ from 'jquery';
 import Button from '@cdo/apps/templates/Button';
 import i18n from '@cdo/locale';
+import ProgressBubbleSet from '@cdo/apps/templates/progress/ProgressBubbleSet';
+import SublevelCard from '@cdo/apps/code-studio/components/SublevelCard';
+import _ from 'lodash';
 
 const VIDEO_WIDTH = 670;
 const VIDEO_HEIGHT = 375;
 const VIDEO_MODAL_WIDTH = 700;
 
+const styles = {
+  sublevelCards: {
+    display: 'flex',
+    flexWrap: 'wrap'
+  }
+};
+
 export default class LevelDetailsDialog extends Component {
   static propTypes = {
     scriptLevel: PropTypes.object.isRequired,
     handleClose: PropTypes.func.isRequired
   };
 
+  constructor(props) {
+    super(props);
+    const scriptLevel = _.cloneDeep(props.scriptLevel);
+    const selectedLevel = scriptLevel.level;
```

Comment:
I think script levels can have more than one level when you use variants. Could that cause an issue here?

Reply:
When we decide which variant to use do we do that on the back end or on the client? If we decide that on the backend, then I think this is okay?

Reply:
I don't know. Hopefully we decide it on the backend but I'm not entirely sure for this code path